src/Discord.Net.Interactions/Utilities/ApplicationCommandRestUtil.cs

```diff
@@ -223,7 +223,7 @@ public static ApplicationCommandProperties ToApplicationCommandProps(this IAppli
                     Name = command.Name,
                     Description = command.Description,
                     IsDefaultPermission = command.IsDefaultPermission,
-                    DefaultMemberPermissions = (GuildPermission)command.DefaultMemberPermissions.RawValue,
+                    DefaultMemberPermissions = command.DefaultMemberPermissions.RawValue == 0 ? new Optional<GuildPermission>() : (GuildPermission)command.DefaultMemberPermissions.RawValue,
```

Comment:
Looking at this more, would it make more sense to overwrite as such?
`DefaultMemberPermissions = command.DefaultMemberPermissions.SanitizeGuildPermissions(),`

The issue opened uses the Optional, which I don't believe is necessary.